### ISSUE TITLE
Return a forget password link for a wrong password

### ISSUE THREAD

**Issue by RamvigneshPasupathy** [eaedk/testing-zindi-package#1]
Will be good to have a forget password link for a wrong password instead of throwing an exception like this

`Exception: [ 🔴 ] {'password': 'wrong password'}`

**Comment by eaedk** [comment]
yeah I see what you mean but maybe the password have often a mistake when we type it the first time ...
